src/gfx/math.hpp

```diff
@@ -11,6 +11,18 @@ inline float degToRad(float v) { return v * degToRadFactor; }
 
 constexpr float radToDegFactor = (180.0f / pi);
 inline float radToDeg(float v) { return v * radToDegFactor; }
+
+inline size_t alignTo(size_t size, size_t alignment) {
```

Comment:
I guess you mean `Round up to the next highest power of 2` from that article. Although I believe that one doesn't allow you to specify the alignment.
Good to know in case we need something similar though

Reply:
Maybe it wasn't there after all.
size = (size + alignment - 1) & ~(alignment - 1) should work for powers of 2. (double check the math if you use it though)